R/unreachable_code_linter.R

```diff
@@ -51,16 +104,37 @@ unreachable_code_linter <- function() {
 
     xml <- source_expression$xml_parsed_content
 
-    bad_expr <- xml_find_all(xml, xpath)
+    lints_return_stop <- xml_find_all(xml, xpath_return_stop)
```

Comment:
nit: elsewhere in the codebase, the flow of logic is rearranged when there's >1 XPath

Here you've done

(1) run all `xml_find_all()`
(2) run all `xml_nodes_to_lints()`

We usually do

(1) Run `xml_find_all()` and `xml_nodes_to_lints()` for XPath 1
(2) ...
(n) Run `xml_find_all()` and `xml_nodes_to_lints()` for XPath n

`package_hooks_linter()` may be a good example.

The result is the same of course, but for consistency I would use the latter pattern.